### PR TITLE
Refactor unneeded 'continue' jumps in tests

### PR DESCRIPTION
**Pull request by eumiro**

### CODE REVIEW

File: tests/always/test_project_structure.py [hunk 1]
```diff
@@ -147,12 +147,9 @@ def get_classes_from_file(self, filepath: str):
...
         for current_node in ast.walk(doc_node):
-            if not isinstance(current_node, ast.ClassDef):
-                continue
-            name = current_node.name
-            if not name.endswith(tuple(self.CLASS_SUFFIXES)):
-                continue
-            results[f"{module}.{name}"] = current_node
+            if isinstance(current_node, ast.ClassDef):
+                if current_node.name.endswith(tuple(self.CLASS_SUFFIXES)):
+                    results[f"{module}.{current_node.name}"] = current_node
```
Comment (hussein-awala) [inline tests/always/test_project_structure.py]: For readability, could you replace inner if by `and` operator?

File: tests/serialization/test_serialized_objects.py [hunk 2]
```diff
@@ -39,10 +39,9 @@ def test_recursive_serialize_calls_must_forward_kwargs():
 
     class_def = None
     for stmt in ast.walk(tree):
-        if not isinstance(stmt, ast.ClassDef):
-            continue
-        if stmt.name == "BaseSerialization":
-            class_def = stmt
+        if isinstance(stmt, ast.ClassDef):
+            if stmt.name == "BaseSerialization":
+                class_def = stmt
```
Comment (hussein-awala) [inline tests/serialization/test_serialized_objects.py]: same here

**Review by potiuk — APPROVED** [review]

**Comment by potiuk** [comment]
Needs some test/stacic checks fixes though

**Comment by eladkal** [comment]
@eumiro can you fix the tests?

**Review by hussein-awala — APPROVED** [review]

**Review by vincbeck — APPROVED** [review]

**Pull request merged**